utils/anautilities.py

```diff
@@ -384,16 +385,11 @@ def getGemType(entry):
     print("Writing output data")
 
     # Write out the dacVal results to a root file, a text file, and the terminal
-    outputTxtFiles_dacVals = ndict()
     for idx in range(len(dacNameArray)):
         dacName = np.asscalar(dacNameArray[idx])
         for entry in crateMap:
             ohKey = (entry['shelf'],entry['slot'],entry['link'])
             detName = getDetName(entry)
```

Comment:
Are these for-loop still necessary? I mean, they only set variables which are redefined in the following for-loops.

Reply:
they are not, thanks for catching this